components/next_steps_view/steps/complete_profile_step/complete_profile_step.tsx

```diff
@@ -19,6 +19,8 @@ import {StepComponentProps} from '../../steps';
 
 import './complete_profile_step.scss';
 
+const MAX_FULL_NAME_LENGTH = 128;
```

Comment:
Should we also limit each first/last string to 64 individually when it's split out (line 88)?

Reply:
Could be confusing given that we're using just one input

Reply:
What is the server behavior if over 64 chars the first/last name? If they input something where the first name is 100 chars and the last name 15, the split will still send the first name too long and will behave the same as the original report for team. 

Maybe we can set a limit when we trim the first/last name by just taking the first 64 of each?

Reply:
Changed. Now is validating first and last name individually

Reply:
> 2. for the name, the error messages shows "Your first name must be less than 65 characters"
> 
> * this is confusing since it's just one 'name' field, not two separate first and last name fields
> * Also, how do we know it's a first or last name? Sometimes a first name has a space in it or sometimes a last name has a space in it, so splitting the name up based on the space character might not always work
>   ![image](https://user-images.githubusercontent.com/2040554/96472874-f8e42a00-11fe-11eb-97a1-6c0e4d4fea03.png)

We are currently taking everything after the first space as the last name. Given the nature of the `Full Name` field we have to make some assumptions. 

I agree that this validation it's confusing and I suggest that we keep the full length restriction as originally implemented and silently trim the first/last names if they exceed 64 chars. 

Thoughts?

Reply:
> 2. for the name, the error messages shows "Your first name must be less than 65 characters"
> 
> * this is confusing since it's just one 'name' field, not two separate first and last name fields
> * Also, how do we know it's a first or last name? Sometimes a first name has a space in it or sometimes a last name has a space in it, so splitting the name up based on the space character might not always work
>   ![image](https://user-images.githubusercontent.com/2040554/96472874-f8e42a00-11fe-11eb-97a1-6c0e4d4fea03.png)

We are currently taking everything after the first space as the last name. Given the nature of the `Full Name` field we have to make some assumptions. 

I agree that this validation it's confusing and I suggest that we keep the full length restriction as originally implemented and silently trim the first/last names if they exceed 64 chars. 

Thoughts?

Reply:
See my comment below. I was suggesting trimming when setting the values and not change validation of the full name.